Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.11 as base
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
+
+RUN source $HOME/.poetry/env && poetry update && poetry install
```

Comment:
Rather than needing to source this every time, we can add poetry to the path, e.g. before installing poetry we specify where to install it with `ENV POETRY_HOME=/poetry` and then after we can add poetry to the path with `ENV PATH=/poetry/bin:${PATH}`